test/check-machines-nics

```diff
@@ -154,12 +155,20 @@ class TestMachinesNICs(VirtualMachinesCase):
         b.wait_text("#vm-subVmTest1-network-1-mac", "52:54:00:4b:73:5f")
         b.wait_text("#vm-subVmTest1-network-2-mac", "52:54:00:4b:73:6f")
         b.wait_text("#vm-subVmTest1-network-3-mac", "52:54:00:4b:73:4f")
+        b.wait_text("#vm-subVmTest1-network-3-model", "virtio")
+        b.wait_text("#vm-subVmTest1-network-4-mac", "52:54:00:4b:73:4f")
+        b.wait_text("#vm-subVmTest1-network-4-model", "e1000e")
         # Detach
         self.deleteIface(2)
         # Make sure popup dialog disappeared
         b.wait_not_present(".pf-c-modal-box")
         b.wait_text_not("#vm-subVmTest1-network-2-mac", "52:54:00:4b:73:6f")
         b.wait_text("#vm-subVmTest1-network-2-mac", "52:54:00:4b:73:4f")
+        b.wait_text("#vm-subVmTest1-network-2-model", "virtio")
+        # Test deleting the interface with same MAC address as the other vNIC will delete the correct one
+        self.deleteIface(2)
+        b.wait_text("#vm-subVmTest1-network-2-mac", "52:54:00:4b:73:4f")
+        b.wait_text("#vm-subVmTest1-network-2-model", "e1000e")
```

Comment:
Ah, so the (previous) third interface gets shifted up to be the second.  Can you please add a `wait_not_present()` assertion that there is no third interface any more?